invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentMetadata/DocumentMetadata.js

```diff
@@ -317,7 +329,7 @@ export default class DocumentMetadata extends Component {
             <Grid.Column>
               <Container>
                 <Header as="h3">Abstract</Header>
-                <p>{document.metadata.abstracts}</p>
+                <p>{document.metadata.abstracts[0].value}</p>
```

Comment:
We can iterate on the abstracts here and display each one in a `p` element.

Reply:
UX is gonna come especially when we have some real data :)